### DISCUSSION TITLE
When will transform and transfromFeedback be reenabled?

### DISCUSSION THREAD

**Discussion by mobhuiyan98** [visgl/luma.gl#2164] [Q&A] [answered]
When will transform and transform Feedback be re-enabled?

**Comment by ibgreen** [comment, marked as answer]
The good news is that the replacements are implemented: https://github.com/visgl/luma.gl/tree/master/modules/engine/src/transform. 

But yes, documentation and examples still need to be updated.

**Reply by ibgreen** [reply]
> Are these deprecated?

Not exactly. But certainly `BufferTransform` is not portable to WebGPU so we either need some higher level abstractions or applications need to do more work.

**Reply by mobhuiyan98** [reply]
I am currently trying to follow a simple example of using the BufferTransform code, I have linked a codesandbox. Is there something that I am missing? 


https://codesandbox.io/p/sandbox/transform-feedback-code-d3hvwh?file=%2Fsrc%2FApp.js

**Reply by ibgreen** [reply]
Can't access, maybe check permissions on your sandbox?

**Reply by mobhuiyan98** [reply]
Should be open now. 

https://codesandbox.io/p/sandbox/transform-feedback-code-d3hvwh

**Reply by ibgreen** [reply]
OK, took a quick peek into BufferTransform source. 

At first blush, it seems clear that the default fragment shader is not configured to match the vertex shader, perhaps it could cause issues, even though we don't care about the output, it should probably reference the same varying to make sure it is not optimized out. You could try to provide your own.

Also worth looking at the BufferTransform test case.
https://github.com/visgl/luma.gl/blob/e6e014efa323dbcd7d3774e982e442b1bf2c93be/modules/engine/test/transform/buffer-transform.spec.ts#L53
